src/components/DropdownMenu.js

```diff
@@ -174,9 +174,4 @@ DropdownMenu.propTypes = {
     ),
 };
 
-DropdownMenu.defaultProps = {
-    size: 'medium',
-    type: 'ghost',
-};
-
```

Comment:
Following the changelog in [pelagos-packages](https://github.com/bluecatengineering/pelagos-packages/blob/master/packages/pelagos/CHANGELOG.md#12120-2024-06-05). I believe we should also eliminate the outdated defaultProps in the Limani project. Because passing defaultProps is no longer necessary, the component will still render correctly.

Reply:
If you remove this from here, please add it in the default props of the component. If we don't add this in the component then in the storybook for Limani it won't show a default value for `size` and `type`